jak-na-konflikty.md

```diff
@@ -0,0 +1,77 @@
+# Jak zvládat konflikty s kolegou?
+
+Cílem této příručky je pomoct ti zvládat konfliktní situace mezi tebou a ostatními Appliftery.
+
+V Kodexu Appliftera se můžeš dočíst, že řešení konfliktů je přirozenou součástí každé organizace. Proto je důležité, abys věděl, jak ke konfliktu přistupovat tak, abys minimalizoval jeho toxicitu pro tebe a pro tvého kolegu a zároveň aby tvé jednání směřovalo k jeho vyřešení.
+
+## Příprava
+
+Před tím, než začneš s kolegou konflikt řešit, je dobré si v hlavě či na papíře ujasnit pár věcí. 
+
+**Cílem přípravy není mít co nejvíc argumentů k tomu, abys svůj protějšek ubil argumenty.** Cílem je schopnost nahlédnout na danou věc z více úhlů, vžít se do svého kolegy a pochopit jeho motivaci a chování.
+
+- Co je věcným "odemočněným" předmětem konfliktu? Jak se daný problém dá popsat co nejvíc věcně?
+- Jak bude předmět konfliktu vnímat a vidět tvůj kolega?
+- Jaké kolem toho máš emoce? Jak se cítíš?
+- Jaké kolem toho bude mít emoce tvůj kolega? 
+- Jaké asi byly motivace kolegy? Proč postupoval tak jak postupoval?
+
+## Před konfliktem
+
+Pokud máš s nadcházejícím konfliktem spojených hodně negativních emocí, zkus je nejdříve zmírnit. Sedni si sám v tiché místnosti na 5 minut a zamedituj si o tvém kolegovi. Soustřeď se na jeho kladné vlastnosti a hezké věci, co jste spolu zažili. Uvědom si, že nacházející konflikt je pravděpodobně něco, čemu se za půl roku nad pivem zasmějete.
+
+## Vyvolání řešení konfliktu
+
+Zastav kolegu v soukromí a popros ho, jestli by si na tebe na chvilku neudělal čas. Řekni mu, že s ním potřebuješ vyřešit nějaké pnutí (klidně to můžeš říct přátelštější či vtipnější formou). Pokud čas nemá, domluv se na čase, kdy se to bude oběma stranám hodit.
```

Comment:
Co když se člověk bojí řešit konflikt s kolegou?

Reply:
Jak píšu výše, už tady by měl do toho vstoupit třetí/další člověk, mediator/někdo z rady.

Reply:
@hauseja3 Sry, asynchronní komentáře 😄

Reply:
@hauseja3 I když tady je ten směr neřešení opačný a s úplně jinou motivací, nechal bych tu diskusi tady a chtěl bych, aby se vyřešila.

Reply:
Když se bojí, rovnou se domluví, že si najdou mediátora. Myslím si, že tím je to elegantně vyřešené. 

Odmítnout řešit konflikt nelze. (Nebo jako lze, ale za mě je to porušení kodexu)

Reply:
@Vratislav když se někdo někoho bojí a proto to s ním nechce řešit, nemůžeme ho k tomu nutit...

Reply:
Pokud to není schopný udělat s mediací, tak pak nevím, pak se to bude muset rovnou řešit s radou starších.

Reply:
@Vratislav no ale co ty říkáš je, že to s tím člověkem má řešit tak jako tak, i když se ho bojí. Za mě je jedno, jestli to je 1:1, s mediátorem nebo s radou. Ten člověk to prostě řešit nebude, protože se bude bát a pak to shoří.

Reply:
@kirscfil tohle je asi na jinej thread.